test/simulations/run-simulation.ts

```diff
@@ -0,0 +1,70 @@
+import { exp, makeProtocol, wait } from '../helpers';
+import { Supplier } from './actors/Supplier';
+import { Actor } from './actors/Actor';
+import { Market } from './Market';
+import { simulate } from './Simulate.ts';
+import { World } from './World';
+import { Borrower } from './actors/Borrower';
+
+// XXX doesn't have to be under the unit test directory. can be a standalone script
```

Comment:
Yeah I guess that probably makes more sense, and we could still import helpers

Reply:
Moved this to be its own script/directory.